src/ensembl/src/content/app/browser/browser-region-field/BrowserRegionField.tsx

```diff
@@ -101,16 +96,31 @@ export const BrowserRegionField = (props: BrowserRegionFieldProps) => {
     }
   };
 
-  const closeForm = () => resetForm();
+  const closeForm = (event: Event) => {
+    if (
+      inputGroupRef?.current?.contains(event.target as HTMLElement) ||
+      buttonRef?.current?.contains(event.target as HTMLElement)
+    ) {
+      return;
+    }
+
+    resetForm();
+  };
+
+  useEffect(() => {
+    document.addEventListener('click', closeForm);
+    return () => document.removeEventListener('click', closeForm);
+  }, []);
```

Comment:
Created a ticket for myself to check if we can update useOutsideClick to intake multiple refs.
https://www.ebi.ac.uk/panda/jira/browse/ENSWBSITES-452